modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -21,16 +21,17 @@ export interface GenerateMpcWalletOptions {
   originalPasscodeEncryptionCode?: string;
   enterprise?: string;
   walletVersion?: number;
-  backupProvider?: string;
+  backupProvider?: BackupProvider;
 }
 
+export type BackupProvider = 'BitGoTrustAsKrs';
```

Comment:
If you're adding better types, can you instead have `BitGoTrustAsKrs` as const type, so we can more in the future. Something similar to what we did for hsm types.
```
export const hsmTypes = ['institutional', 'nitro'] as const;

export const HSMTypes = createStringObject(hsmTypes);
export type HSMType = typeof hsmTypes[number];
```

Reply:
This is not trivial to do because there is no util `createStringObject` in the sdk